src/pipeline.cc

```diff
@@ -206,6 +207,12 @@ class PipelineWorker : public AsyncWorker {
         RemoveExifOrientation(image);
       }
 
+      // Trim
+      // TODO(kleisauke): Not sure if we should check if accesMethod is VIPS_ACCESS_SEQUENTIAL
```

Comment:
How about adding logic to `NAN_METHOD(pipeline)` that when `trimTolerance` is set then we also force the `accessMethod` to `VIPS_ACCESS_RANDOM`?